i3pystatus/core/io.py

```diff
@@ -50,24 +52,37 @@ class StandaloneIO(IOHandler):
     and the i3bar protocol header
     """
 
+    __is_click_event = False
```

Comment:
Identifiers starting with `__` shouldn't be normally used (per PEP8) ; use `_` instead.